app/api/listicle/email_links.rb

```diff
@@ -0,0 +1,10 @@
+module Email_Links
```

Comment:
This namespace is a little off. In ruby, it's conventional to use CamelCase (as opposed to  Screaming_Snake_Case) for module and class names. Also, you should use modules to reflect the namespace of your directory structure. In this case it would be Listicle.

IMHO, Listicle isn't a great descriptor of what this is (I know the reference app uses it all over the place; I grew out of favor with it pretty quickly).

I think you can remove the listicle directory, move this file up into the api directory and rename it to `email_link.rb` and rename the class `EmailLink`. Then, you can make the resource `:email_link` and make it all consistent.